source/vocab/display.jsonld

```diff
@@ -150,12 +162,24 @@
           "classLensDomain": "Country",
           "showProperties": [ "prefLabel" ]
         },
+        "marc:AddedEntryHierarchicalPlaceName": {
+          "@id": "AddedEntryHierarchicalPlaceName-chips",
+          "@type": "fresnel:Lens",
+          "classLensDomain": "marc:AddedEntryHierarchicalPlaceName",
+          "showProperties": [ "countryOrLargerEntity", "city" ]
+        },
```

Comment:
I figure this was necessary for some operative requirement? (Since we avoid marc-terms, especially classes, here.)

Reply:
Yes for older material where name of the places have shifted over time this is an important field. It is also a reminder for us that we want linked geodata ;)

Reply:
Fair game then!